portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -28,13 +28,20 @@
   private final ArrayList<String> msgs = new ArrayList<String>();
 
   @Override
-  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    msgs.add("hello");
-    msgs.add("there");
-    msgs.add("world");
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {    
     Gson gson = new Gson();
     String json = gson.toJson(msgs);
+    // Respond with the result
```

Comment:
I think this comment may be unnecessary. The next line is pretty self-explanatory given the descriptive function names.